docs/Guidelines.md

```diff
@@ -19,22 +19,18 @@ Please use [rebase instead of merge](https://www.atlassian.com/git/tutorials/mer
 
 ### Commit Message Style
 
-#### Commit Summary
+This project uses the [Conventional Commits specification](https://www.conventionalcommits.org/) to ensure a consistent way of dealing with commit messages.
 
-To be decided.
+````
+feat(formatter/json.ts): Adds multi UI5 version support
```

Comment:
As a reader of the changelog I wouldn't be interested in the actual source file name. If it is a module that can be imported via Node.js (e.g. (in theory) `import json from "@ui5/linter/formatter/json"`), then that's the important information I would like to see.

Reply:
And as a developer/maintainer, I can just look at the commit and see the changed files.

Reply:
Adjusted the example with a real word commit.